migrations/0037-add-slides-link-to-events-table.ts

```diff
@@ -0,0 +1,17 @@
+import { MigrationInterface, QueryRunner, TableColumn } from 'typeorm';
+
+const TABLE_NAME = 'Events';
+
+export class AddSlidesLinkToEventsTable1696465672671 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.addColumn(TABLE_NAME, new TableColumn({
+      name: 'slidesLink',
+      type: 'text',
```

Comment:
Is there a specific reason you opted for "text" instead of "varchar(255)"? According to [this](https://www.postgresql.org/docs/current/datatype.html) documentation, text and varchar have the same description. But since we are consistent with varchar everywhere else, I believe it would be better to put it here as well.